Modules/System/Email/src/Email/EmailImpl.Codeunit.al

```diff
@@ -437,4 +449,27 @@ codeunit 8900 "Email Impl"
             Error(InsufficientPermisionsErr);
     end;
 
+    local procedure GetRecordRefFromVariant(RecRelatedVariant: Variant; var ResultRecordRef: RecordRef): Boolean
```

Comment:
@mazhelez Is there a generic place where this method can be added? I've seen that there are already several copies of this function along the different apps.

Reply:
No, unfortunately, there is no generic place for such helper functions.
Technically, it could be a separate module dedicated to working with RecRef-s.

However, this is an overkill at this point. Just make sure the code isn't duplicated in the current module :)